test/views/nextUpdates/NextUpdatesOptOutViewSpec.scala

```diff
@@ -17,6 +17,7 @@
 package views.nextUpdates
```

Comment:
There is a 

`obligationsMessages` object can this be capitalised and moved into the messages package in the test/views/messages directory?